modules/apps/stat-parser.js

```diff
@@ -188,7 +188,7 @@ export default class StatBlockParser extends FormApplication {
             catch {}
             if (!skillItem) {
                 console.error("Could not find " + skill)
-                ui.notifications.error(game.i18n.format("ERRORParser", {name: skill}), { permanent: true })
+                ui.notifications.error(game.i18n.format("ERROR.Parser", {name: skill}), { permanent: true })
```

Comment:
I've updated the stat parser so pull from develop, the conflicts should be simple to resolve.